source/comply/legal-hold.rst

```diff
@@ -43,86 +43,65 @@ Establish a policy for when to implement a legal hold. This should be developed
-    - Support for additional file storage options such as MinIO or local file storage are under consideration.
-    - You may also optionally use our API to preserve data for legal hold. See our :ref:`electronic discovery <comply/electronic-discovery:mattermost restful api>` product documentation to learn more.
+1. Log in to your Mattermost workspace as a system administrator.
+2. Download the latest version of the `plugin binary release <https://github.com/mattermost/mattermost-plugin-legal-hold/releases/>`_, compatible with Mattermost v8.0.1 and later. If you are using an earlier version of Mattermost, :doc:`follow our documentation </upgrade/upgrading-mattermost-server>` to upgrade to Mattermost v8.0.1 or later.
+3. Go to **System Console > Plugins > Plugin Management > Upload Plugin**, and upload the plugin binary you downloaded in the previous step.
+4. In the **Installed Plugins** section, scroll to **Legal Hold Plugin**, and select **Enable**.
 
-2 - Enable compliance exports
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Configure the plugin
+^^^^^^^^^^^^^^^^^^^^^
 
-Enable compliance exports containing all messages and file uploads in an XML format to your AWS S3 file storage.
-
-The exports include information on channel member history at the time the message was posted, which is critical information for some legal hold use cases. Entries for deleted messages and files are also included.
-
-1. In Mattermost, go to **System Console > Compliance > Compliance Export**.
-2. Set **Enable Compliance Exports** to **true**.
-3. Set the **Compliance Export time**. Choose a time when fewer people are using your system, such as 2:00AM your local time.
-4. Set the export file format to **Actiance XML**.
-5. Learn more about compliance export configuration options in our :doc:`product documentation </comply/compliance-export>`.
+When the Legal Hold integration is enabled, you can configure when it runs in the format ``HH:MM ±HHMM`` and ``+0000`` for UTC, and can optionally configure a custom Amazon S3 bucket for legal holds by specifying Amazon S3 configuration settings. Alternatively, you can use existing :ref:`Amazon S3 file storage <configure/environment-configuration-settings:amazon s3 bucket>` for legal holds, where access keys are supported when using IAM roles.
 
 .. note::
 
-    - For legal hold, only XML format is supported at this time. 
-    - Support for additional file formats, such as CSV or EML, are under consideration.
-
-3 - Leverage compliance API in your DLP tool of choice
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  Only Amazon S3 is supported for Mattermost legal holds at this time, for increased reliability, compliance, and automation capabilities. Support for additional file storage options such as MinIO or local file storage are under consideration. Learn more about file storage configuration options in our :ref:`product documentation <configure/environment-configuration-settings:file storage>`.
```

Comment:
This paragraph makes me think that only S3 is allowed for legal holds to store data, but if you have local storage set up on your Mattermost server and do not set up the S3 connection in the plugin, the legal holds will end up in the local storage.

cc: @wiggin77